content/en/docs/reference/glossary/kube-scheduler.md

```diff
@@ -10,14 +10,52 @@ aka:
 tags:
 - architecture
 ---
-Control plane component that watches for newly created
-{{< glossary_tooltip term_id="pod" text="Pods" >}} with no assigned
-{{< glossary_tooltip term_id="node" text="node">}}, and selects a node for them
-to run on.
+A control plane component that watches 
```

Comment:
If we are improving the longer versions of the glossary, I think it also makes sense to update the `short_description` in the front matter at the top of the file to use the same wording.